added cards for ALPtoTTbar: pure ALP signal and ALP-SM interference

jabuschh opened the pull request:
Dear GEN experts,
I added the cards for ALPs decaying to top quarks pairs for the two cases of pure ALP signal and pure ALP-SM interference. We target a search for non-resonant contributions from ALPs to the ditop mass spectrum.
Thanks in advance for your effort!

sihyunjeon commented:
Hi, one naive question,
how do you treat the top decays? through pythia?

sihyunjeon commented on bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ALPtoTTbar/ALP_ttbar_interference/ALP_ttbar_interference_run_card.dat:
```diff
@@ -0,0 +1,274 @@
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  100 = nevents ! Number of unweighted events requested
```
change this to 3000, pilot run 100 is a bit small

jabuschh replied:
Okay, I think this here was just the default value in my madgraph card.
I'm no expert, but won't this be overwritten later anyways?
We would like target 10M events for each sample (signal, interference) and I will contact our b2g MC experts on this.

agrohsje replied:
No impact on production. Just needed for the pilot run so you get enough stats to have a meaningful test.

jabuschh replied:
Alright, thanks for the info, then I will change this value to 3000 in both cases

sihyunjeon commented on bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ALPtoTTbar/ALP_ttbar_signal/ALP_ttbar_signal_proc_card.dat:
```diff
@@ -0,0 +1,15 @@
+define l- = e- mu- ta-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+generate p p > t t~ NP==2
```
Does NP==2 work?
Should not it be NP=2 instead?

jabuschh replied:
Yes, this works. I did extensive studies on this.
If you want, you can have a look at my public area on lxplus: /afs/cern.ch/user/h/hjabusch/public/ALPToTTbar_genproduction
There you can find the gridpacks, log files, customizecards and also the full output directories from madgraph.

jabuschh commented:
Hi, yes indeed, we will use Pythia8 for the hadronization

jabuschh commented:
In addition, I already did some fast sim studies using Pythia and Delphes on these ALP samples and we are also in constant exchange with the authors of this model. Therefore, we are confident that these cards are reasonable.
If you have further questions, please let me know!

agrohsje commented:
I see in your logs that you used 2.7.2 and patches failed. Will you use master for official production. Note that we didn't validate 2.7.2 for official production.

jabuschh commented:
Good point, I didn't know that. I now changed the number of pilot run events to 3000 (see latest commit), used the setup from the master branch (v2.6.5) and updated the files in here: /afs/cern.ch/user/h/hjabusch/public/ALPToTTbar_genproduction.
Now it looks also better for the patches. For official production we will stick to this version of course.

agrohsje commented:
Happy to merge the cards. @sihyunjeon let me know if you agree?

sihyunjeon commented:
meged

jabuschh commented:
Perfect, thank you!
